docs/aced.md

```diff
@@ -463,6 +463,44 @@ environment:
 revproxy-service:
 ```
 
+## Change the data dictionary
+
+* Update the data dictionary.  In this case we've updated it from the data_dictionary repo.
+  * Note: This is not the only potential source of changes, see https://github.com/uc-cdis/compose-services/blob/master/docs/using_the_commons.md#changing-the-data-dictionary
+
+```commandline
+  cp ../data_model/DATA/gen3/*.yaml datadictionary/gdcdictionary/schemas/
+  #  "compile" the yaml into a single json file
+  python3 etl/compile.py datadictionary/gdcdictionary/schemas/  --out datadictionary/
+  #  copy the resulting datadictionary/aced.json  to S3
+  # verify you can read https://aced-public.s3.us-west-2.amazonaws.com/aced.json
+```
+ 
+* Set the new datadictionary URL in docker-compose
+```commandline
+git diff docker-compose.yml 
+diff --git a/docker-compose.yml b/docker-compose.yml
+index 7d139a0..6280178 100644
+--- a/docker-compose.yml
++++ b/docker-compose.yml
+@@ -125,7 +125,7 @@ services:
+       - ./scripts/peregrine_setup.sh:/peregrine_setup.sh
+       - ./datadictionary/gdcdictionary/schemas:/schemas_dir
+     environment: &env
+-      DICTIONARY_URL: https://s3.amazonaws.com/dictionary-artifacts/datadictionary/develop/schema.json
++      DICTIONARY_URL: https://aced-public.s3.us-west-2.amazonaws.com/aced.json
+
+```
+
+* add custom graphql section to 
```

Comment:
Where should the custom graphql section be added to?

Reply:
See https://github.com/ACED-IDP/compose-services-training/pull/8/commits/9472acf9229bff6cb2fa14b81d7ae5312304529f#diff-974fe187f1de7376743d2cfebeeebe63729d4d409bbe7077d048e185dacebba3R509

Reply:
see examples/Secrets/gitops.json